tools/ci_connector_ops/ci_connector_ops/utils.py

```diff
@@ -1,14 +1,19 @@
 #
 # Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 #
+
+
+from dataclasses import dataclass
 import logging
+from pathlib import Path
 from typing import Dict, Optional, Set, Tuple
 
 import git
 import requests
 import yaml
 
 AIRBYTE_REPO = git.Repo(".")
+DIFFED_BRANCH = "origin/master"
```

Comment:
I declared this as a constant for easier mocking. When running test we want to switch it to the active branch.